src/Core/src/PublicAPI/net-android/PublicAPI.Shipped.txt

```diff
@@ -2068,7 +2068,7 @@ override Microsoft.Maui.Animations.LerpingAnimation.Update(double percent) -> vo
 override Microsoft.Maui.Animations.PlatformTicker.IsRunning.get -> bool
 override Microsoft.Maui.Animations.PlatformTicker.Start() -> void
 override Microsoft.Maui.Animations.PlatformTicker.Stop() -> void
-override Microsoft.Maui.Animations.PlatformTicker.SystemEnabled.get -> bool
+
```

Comment:
For the tools to make a nice diff, this has to be done via removing it in the Unshipped.